schema/encoding/jsonschema.go

```diff
@@ -139,6 +140,13 @@ func schemaToJSONSchema(w io.Writer, s *schema.Schema) (err error) {
 		if ew.err == nil {
 			ew.err = validatorToJSONSchema(w, field.Validator)
 		}
+		if field.Default != nil {
+			b, err := json.Marshal(field.Default)
+			if err != nil {
+				return err
+			}
+			ew.writeFormat(`, "default": %s`, string(b))
```

Comment:
Yeah this works:-) Can be optimized later by avoiding the conversion from []byte -> string -> []byte, but except for that this should be correct:-) Thanks for adding it.